vpp-manager/vpp_runner.go

```diff
@@ -808,6 +864,9 @@ func (v *VppRunner) runVpp() (err error) {
 		return errors.Wrap(err, "Error configuring VPP")
 	}
 
+	// add main network that has the default VRF
+	config.Info.PhysicalNets[""] = config.PhysicalNetwork{VrfId: common.DefaultVRFIndex, PodVrfId: common.PodVRFIndex}
```

Comment:
Might be worth adding a `const DefaultPhysicalNetworkName = ""` just to make the code clearer e.g.
````golang
config.Info.PhysicalNets[DefaultPhysicalNetworkName] = ...
````